stats/latencyhistogram15s32.go

```diff
@@ -30,8 +33,9 @@ func (l *LatencyHistogram15s32) ReportGraphite(prefix, buf []byte, now time.Time
 	// for now, only report the summaries :(
 	r, ok := l.hist.Report(snap)
 	if ok {
+		sum := atomic.SwapUint64(&l.sum, 0)
```

Comment:
you are reseting sum to 0 on every flush, but not count.

Reply:
r.Count comes from the report which comes from the histogram snapshot. the histogram is reset when snapshot is taken